rclcpp/test/rclcpp/test_clock.cpp

```diff
@@ -41,9 +42,10 @@ class TestClockWakeup : public ::testing::TestWithParam<rcl_clock_type_e>
         clock->sleep_until(clock->now() + std::chrono::seconds(3));
         thread_finished = true;
       });
-
+    RCPPUTILS_DEPRECATION_WARNING_OFF_START;
     // notify the clock, that the sleep shall be interrupted
     clock->cancel_sleep_or_wait();
+    RCPPUTILS_DEPRECATION_WARNING_OFF_STOP
```

Comment:
semi-colon required or free? i think either is fine but we would want it to be consistent?

```suggestion
    RCPPUTILS_DEPRECATION_WARNING_OFF_START;
    // notify the clock, that the sleep shall be interrupted
    clock->cancel_sleep_or_wait();
    RCPPUTILS_DEPRECATION_WARNING_OFF_STOP;
```